src/main/kotlin/lotto/domain/Lottos.kt

```diff
@@ -19,7 +19,16 @@ class Lottos(private val tickets: List<Lotto>) {
     ): Map<Rank, Int> {
         return tickets
             .map { it.match(winningNumber, bonusBall) }
-            .groupBy { it }
-            .mapValues { it.value.size }
+            .groupingBy {
+                it
+            }.eachCount()
+    }
+
+    operator fun plus(lotto: Lottos): List<Lotto> {
+        return tickets + lotto.tickets
+    }
```

Comment:
아래와 같이 리턴 타입을 Lottos로 변경해보면 어떨까요? 😃

```suggestion
    operator fun plus(lotto: Lottos): Lottos {
        return Lottos(tickets + lotto.tickets)
    }
```